.yamllint.yml

```
@@ -0,0 +1,46 @@
+# YAML linting configuration
+extends: default
+
+rules:
+  line-length:
+    max: 120
+    level: warning
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)